src/components/molecules/SfGallery/SfGallery.js

```diff
@@ -0,0 +1,64 @@
+import Glide from "@glidejs/glide";
+import lozad from "lozad";
+
+export default {
+  props: {
```

Comment:
please document props so it's visible on storybook tables (see other components for examples)